### PR TITLE
Support for device events on group broadcast & device link cache

### PR DESCRIPTION
**Pull request by PaulWieland**

### CODE REVIEW

File: src/PowerLincModem.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import logger from 'debug';
+/* Configuring logging */
+const debug = logger('node-red-contrib-insteon:powerLincModem');
```
Comment (apearson) [inline src/PowerLincModem.ts:3]: This references `node-red-contrib-insteon` but this is the plm library.

File: src/PowerLincModem.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+debug.enabled = true;
```
Comment (apearson) [inline src/PowerLincModem.ts:4]: Debug enabled by default?

**Review by apearson — APPROVED** [review]

File: src/PowerLincModem.ts (hunk 1 above)
Comment (PaulWieland) [reply src/PowerLincModem.ts:3]: Mistake. Should be `insteon-plm`.

File: src/PowerLincModem.ts (hunk 2 above)
Comment (PaulWieland) [reply src/PowerLincModem.ts:4]: Also a mistake.

**Pull request merged**